assets/js/search.js

```diff
@@ -8,80 +8,98 @@ async function searchMusic() {
       "<p class=''>Please enter a song or artist's name.</p>";
     return;
   }
+
   const itunesResponse = await fetch(
     `https://itunes.apple.com/search?term=${encodeURIComponent(
       searchInput
-    )}&entity=song&limit=100`
+    )}&entity=song&limit=25`
```

Comment:
Makes sense. Maybe we should add logic to also go to another page once we scroll through the 25 results?